products/storage/api.yaml

```diff
@@ -687,5 +700,70 @@ objects:
         values:
           - :OWNER
           - :READER
+  - !ruby/object:Api::Resource
+    name: 'Object'
+    base_url: b/{{bucket}}/o/{{object}}
+    self_link: b/{{bucket}}/o/{{object}}
+    description: |
+      Information about an object stored in a GCS bucket
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'bucket'
+        description: 'The name of the bucket.'
+        required: true
+      - !ruby/object:Api::Type::String
+        name: 'object'
+        description: 'The name of the object.'
+        required: true
+      - !ruby/object:Api::Type::String
+        name: 'contentType'
```

Comment:
Is there a list of possible content types? It would be really nice for users to know what values are possible to expect.

Doesn't have to be an enum, but a link would be helpful (if it exists)

Reply:
Totally!